catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -24,23 +26,29 @@
 
     public static final String XML_BOUNDS_TYPE = "GM_Envelope";
 
+    public static final String REGISTRY_ID_PROPERTY = "org.codice.ddf.registry.registry-id";
+
     //ebrim constants
     //Object types
     public static final String REGISTRY_NODE_OBJECT_TYPE = "urn:registry:federation:node";
 
     public static final String REGISTRY_SERVICE_OBJECT_TYPE = "urn:registry:federation:service";
 
-    public static final String REGISTRY_CONTENT_COLLECTION_OBJECT_TYPE =
-            "urn:registry:content:collection";
+    public static final String REGISTRY_CONTENT_COLLECTION_OBJECT_TYPE = "urn:registry:content:collection";
 
     //object ids
-    public static final String REGISTRY_MCARD_LOCAL_ID = "urn:mcard:local-id";
+    public static final String REGISTRY_MCARD_ID_LOCAL = "urn:mcard:local-id";
```

Comment:
Will need to change quite a few test files to make this change but I think it is probably worth it to make things consistent with other urn's in the system.